src/java/org/apache/cassandra/streaming/async/StreamingInboundHandler.java

```diff
@@ -262,7 +254,7 @@ else if (message instanceof IncomingStreamMessage)
             if (streamSession == null)
                 throw new IllegalStateException(createLogTag(null, channel) + " no session found for message " + message);
 
-            streamSession.attach(channel);
+            streamSession.attachInbound(channel);
```

Comment:
I didn't add `isControlConnect` param to `attachInbound`, because:
* on initiator side, the control channel has already been created; 
* on follower side, the control channel is attached when receiving `StreamInitMessage`.

I will include it to improve clarity..